### PR TITLE
add laneware `raindrop` keyboard

### PR DESCRIPTION
**Pull request by dunk2k**
## Description

During a request for help in r/qmk; discovered this board's firmware files need updated as they fail build with current version of QMK.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by sigprof** [comment]
Looking at the [shop page](https://lanewareperipherals.com/collections/raindrop-60), it appears that there is only a single version of the PCB that supports multiple layouts.  The way to support such PCBs in QMK is to add a single keyboard with multiple entries under `layouts` in `info.json`, not to create multiple separate keyboards for each possible layout.

**Comment by dunk2k** [comment]
> Looking at the [shop page](https://lanewareperipherals.com/collections/raindrop-60), it appears that there is only a single version of the PCB that supports multiple layouts. The way to support such PCBs in QMK is to add a single keyboard with multiple entries under `layouts` in `info.json`, not to create multiple separate keyboards for each possible layout.

Appears this reference on [geekhack.org](https://geekhack.org/index.php?topic=105885.0) is more accurate than the shop page.

**Comment by sigprof** [comment]
The [GB thread](https://geekhack.org/index.php?topic=107163) claims that ISO is supported, but still shows the same PCB image without any ISO support; also all renders and prototypes show a plate with a single supported layout (ANSI with 2u left Shift and cursor keys).  No idea how the actual GB boards look.

**Review by sigprof — CHANGES_REQUESTED** [review]

File: keyboards/raindrop/info.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "backlight": false,
+        "audio": false,
+        "rgblight": false,
+        "sleep_led": false
+    },
+    "community_layouts": ["60_hhkb", "60_ansi"],
+    "layouts": {
```
[the remark this answers is not in the document]
Comment (dunk2k) [reply keyboards/raindrop/info.json:31]: LAYOUT_all added

**Comment by dunk2k** [comment]
@Laneware 
Could you review to ensure this PR matches the Group Buy board being added

**Review by drashna — APPROVED** [review]

File: keyboards/raindrop/readme.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    make raindrop:default
+    make raindrop:arrows
+    make raindrop:hhkb
+
+Flashing example for this keyboard:
+
+    make raindrop:default:flash
+    make raindrop:arrows:flash
+    make raindrop:hhkb:flash
```
Comment (fauxpark) [inline keyboards/raindrop/readme.md:32]:
```suggestion

Flashing example for this keyboard:

    make raindrop:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**